File: web-front/src/main/java/com/isystk/sample/web/front/controller/api/v1/upload/UploadRestDto.java
package com.isystk.sample.web.front.controller.api.v1.upload;

import com.isystk.sample.common.dto.Dto;

public class UploadRestDto implements Dto {

  Integer imageId;

  Integer imagePath;

  public Integer getImageId() {
    return this.imageId;
  }

  public Integer getImagePath() {
    return this.imagePath;
  }

  public void setImageId(Integer imageId) {
    this.imageId = imageId;
  }

  public void setImagePath(Integer imagePath) {
    this.imagePath = imagePath;
  }
}

File: web-front/src/main/kotlin/com/isystk/sample/web/front/controller/api/v1/upload/UploadRestDto.kt
package com.isystk.sample.web.front.controller.api.v1.upload

import com.isystk.sample.common.dto.Dto
import lombok.Getter
import lombok.Setter

@Setter
@Getter
class UploadRestDto : Dto {
    var imageId: Int? = null
    var imagePath: Int? = null
}